managed-databases/document-database/reference-content/overview.mdx

```diff
@@ -0,0 +1,48 @@
+---
+meta:
+  title: Managed Document Databases Overview
+  description: This page includes a detailed explanation of the Scaleway Document Database product.
+content:
+  h1: Managed Document Databases Overview
+  paragraph: This page includes a detailed explanation of the Scaleway Document Database product.
+tags: databases managed-databases document mongodb ferretdb
+categories:
+  - managed-databases
+---
+
+FerretDB is an open-source database engine that provides a MongoDB®-compatible API layer over a PostgreSQL engine to store and retrieve data.
+
+The engine allows the query of document data through a NoSQL, JSON-structured query language.
+
+FerretDB relies on a modular architecture built on top of PostgreSQL. Documents are stored in binary JSON format using the corresponding PostgreSQL column type. FerretDB also uses several tables in PostgreSQL to store database structure information such as database names, collection names, and user access management.
+
+FerretDB translates MongoDB® wire protocol queries to SQL to be remain compatible with existing software and tools built on MongoDB®.
+
+<Lightbox src="scaleway-ferretdb-schema.webp" alt="" />
+
+The image above depicts a common use-case of Scaleway Managed Document Databases. Managed Document Databases are hosted on Instances running a PostgreSQL Agent and a FerretDB container.
+
+In this example, the Database Instance is attached to a Private Network. High Availability is also activated, meaning that data is streamed continuously from the Database Instance to a secondary node. In the examlpe, Database Instance configuration is stored on a specific distributed storage, and [heartbeats](/managed-databases/document-database/concepts#heartbeats) are continuously performed to monitor the Instance's status and redirect traffic automatically if an incident is detected on the primary Database Instance. Finally, the Database Instance data is stored on Block Storage.
+
+## Contributing to FerretDB
+
+During the Public Beta phase of Managed Document Databases, you can contribute to FerretDB's open-source project by anonymously sharing the telemetry data of your FerretDB Database Instances.
+
+### Collected data
+
+The data that is collected is anonymous, and is restrained to:
```

Comment:
I wonder if the best place for this detailed content is in the overview (compared to a link to a FAQ entry), but I guess it's because we don't plan having a FAQ entry for Document Database at this stage (which is completely ok for now obviously)

Reply:
I think for now we can keep it here, then when we have an FAQ we can move it there. However if you already have questions in mind we can create an FAQ page, no problem

Reply:
I agree, we will switch it to a FAQ when needed, as there are not yet enough questions/answers to be added to a FAQ.